diff --git a/fastapi/__init__.py b/fastapi/__init__.py
index e672ec9..36ee089 100644
--- a/fastapi/__init__.py
+++ b/fastapi/__init__.py
@@ -1,6 +1,6 @@
 """FastAPI framework, high performance, easy to learn, fast to code, ready for production"""
 
-__version__ = "0.115.14"
+__version__ = "0.115.14+partyops.1"
 
 from starlette import status as status
 
diff --git a/pyproject.toml b/pyproject.toml
index ed47d9b..21eb7b7 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -45,11 +45,11 @@ classifiers = [
     "Topic :: Internet :: WWW/HTTP",
 ]
 dependencies = [
-    "starlette>=0.40.0,<0.47.0",
+    "starlette==1.3.1+partyops.1",
     "pydantic>=1.7.4,!=1.8,!=1.8.1,!=2.0.0,!=2.0.1,!=2.1.0,<3.0.0",
     "typing-extensions>=4.8.0",
 ]
-version = "0.115.14"
+version = "0.115.14+partyops.1"
 
 [project.urls]
 Homepage = "https://github.com/fastapi/fastapi"
